schemas/commentary.js

```diff
@@ -0,0 +1,50 @@
+export default {
```

Comment:
Let's call this as `GitaCommentary` instead of just `commentary` since it is related to mainly `gita-verse`